src/utils.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module wordcount/utils
+ */
+
+/**
+ * Function walks through all the model's nodes. It obtains a plain text from each {@link module:engine/model/text~Text}
+ * and {@link module:engine/model/textproxy~TextProxy}. All sections, which are not a text, are separated with a new line (`\n`).
+ *
+ * **Note:** Function walks through the entire model. There should be considered throttling during usage.
+ *
+ * @param {module:engine/model/node~Node} node
```

Comment:
It actually needs to be `Item` because it's also called with `Text` and `TextProxy`.

But, at the same time, you're completely right that neither of them have `getChildren()`. Which means that if we'd use a statically typed language we'd need to typecast here or solve it differently. That's why I wrote in the ticket that you reported that we should wait with such polishing until we use TS.

Reply:
It looks so - I wrote that ticket after this review actually. I worry that we can have more such mistype declaration in the docs for `Node`/`Element`/`Item` trio.